### PR TITLE
add alternative language folders

### PR DESCRIPTION
**Pull request by theo-chiu**

### CODE REVIEW

**Review by jonmereria-ikigai — DISMISSED** [review]

**Review by jonmereria-ikigai — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,70 +1,20 @@
 # Time Deposit Refactoring Kata
-
 ## XA bank time deposit
 
-It's unfortunate that the junior developer has left the company without handing his work over to anyone. And what is even more unfortunate is that you have been assigned to take over the code that he left behind.
+It's unfortunate that the junior developer has left the company without handing his work over to anyone. And what is even more unfortunate is that you have been assigned to take over the code that he left behind. 
 
 We were able to savage only a portion of the business requirements, but we're not sure if the existing implementation has covered all the flows.
```
Comment (jonmereria-ikigai) [inline README.md:6]: salvage?

**Review by adam-lin-dragon — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (theo-chiu) [reply README.md:6]: there's just 1 extra space, the paragraph is still there

Comment (jonmereria-ikigai) [reply README.md:6]: Sorry, I meant to say 'salvage' instead of 'savage'.

Comment (theo-chiu) [reply README.md]: fixed

**Review by jonmereria-ikigai — APPROVED** [review]

**Review by jonmereria-ikigai — APPROVED** [review]
thanks!

**Review by calvin-luk-ikigai — APPROVED** [review]

**Pull request merged**